README.Rmd

```diff
@@ -5,8 +5,6 @@ output: github_document
 
 `CuratedAtlasQuery` is a query interface that allow the programmatic exploration and retrieval of the harmonised, curated and reannotated CELLxGENE single-cell human cell atlas. Data can be retrieved at cell, sample, or dataset levels based on filtering criteria. 
 
-# Query interface
```

Comment:
Does this `README.Rmd` need to exist? It's just an artifact of knitting the vignette, no?

Reply:
I have splitted  README.Rmd (used for the README) and Introduction.Rmd for the veginette. As their image paths are different, and it is easier to manage now.

I usually include README.Rmd in the git repo.

Reply:
Ah okay.

Reply:
I would be a bit wary about redundancy between these two files. e.g. if we change the API we will now have to change the examples in both of these `.Rmd` files.

Reply:
Yes this is a discussion I have being having for the last 3 years, and Bioconductor might have an elegant solution that I am not aware.

But I would say this redundancy is not that bad, and we can focus our energies elsewhere for the moment, leaving these details down the line.